components/form_inputs/textareas/LoadingTextarea.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <v-container>
+    <v-textarea
+      color="black"
+      label="Label"
+      loading
+      loader-height="10"
+      type="number"
+      hide-spin-buttons
```

Comment:
I'm not sure that it works correct (it definitely works with v-text-field, but with v-textarea I can't reproduce)
Please recheck it

Reply:
I deleted these properties and created a bug report for Vuetify https://github.com/vuetifyjs/vuetify/issues/15997

Reply:
The answer was the v-textarea element doesn't support property type, so it can be promoted without these properties